src/showstyle.sh

```diff
@@ -2,6 +2,18 @@
 #
 # Print the formatting style metrics for the specified directory
 #
+if ! [ -x "$(command -v qmcalc)" ]; then
```

Comment:
the `command` built-in is AFAIK a *bash* feature.  Simply testing a null command invocation is simpler and more comprehensive: `if qmcalc </dev/null >/dev/null ...`